### PR TITLE
added  task #1

### PR DESCRIPTION
**Pull request by nomad898**
Hi, Aliaksandra. Please, review my PR

### CODE REVIEW

File: task1/src/components/HelloWorldCreateElement.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const HelloWorldCreateElement = React.createElement(
+    'div',
+    {},
+    React.createElement('h1', {}, 'Hello World!')
+);
+
+export default HelloWorldCreateElement
```
Comment (aliaksandra-daniliuk) [inline task1/src/components/HelloWorldCreateElement.js:8]: `HelloWorldCreateElement` is never used. Is it ok?)

Comment (nomad898) [reply task1/src/components/HelloWorldCreateElement.js:8]: Hi! Thank you for your reply! Actually, i tried to use it in index.js file. But there is only one ReactDOM.render is available. Or should I wrap it to function?  image

File: task1/src/components/HelloWorldLambdaFunctional.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const HelloWorldLambdaFunctional = (props) => {
+    return (
```
Comment (DrDiman) [inline task1/src/components/HelloWorldLambdaFunctional.js:2]: JFYI: 
arrow functions can be used without `return` as: 
`const HelloWorldLambdaFunctional = (props) => ()`

File: task1/src/components/HelloWorldComponent.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    render() {
+        return (
+            <div>
+                <h1>Hello World! {this.props.someText}</h1>
+            </div>
+        )
+    }
+}
+
+export default HelloWorldComponent
```
Comment (DrDiman) [inline task1/src/components/HelloWorldComponent.js:13]: it's ok export, but use plz named export everywhere as 
`export { HelloWorldComponent }` it's more strict for imports

File: task1/src/components/HelloWorldFunctional.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function HelloWorldFunctional(props) {
+    return (
+        <div>
+            <h1>Hello World! {props.someText}</h1>
+        </div>
+    );
+}
+
+export default HelloWorldFunctional
+
```
Comment (DrDiman) [inline task1/src/components/HelloWorldFunctional.js:10]: do not leave empty lines

File: task1/src/components/HelloWorldCreateElement.js (hunk 1 above)
Comment (aliaksandra-daniliuk) [reply task1/src/components/HelloWorldCreateElement.js:8]: Thank you for answer, I think no need to change anything considering it is test app, everything is top.

File: task1/src/App.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        >
+          Learn React
+        </a>
+      </header>
+      <div>
+        <HelloWorldLambdaFunctional someText="This is Lambda function"/> 
+        <HelloWorldFunctional someText="This is function"/>
+        <HelloWorldComponent someText="This is Component"/>
+        <HelloWorldPureComponent someText="This is Pure Component"/> 
+        <HelloWorldCreateElementFunction />
```
Comment (DrDiman) [inline task1/src/App.js:31]: u can pass props as well)

**Pull request merged**